action.rb

```diff
@@ -1,25 +1,35 @@
 class Action
   attr_reader :headers, :body, :request
 
-  def initialize &block
+  def initialize(&block)
     @block = block
-    @status = 200
-    @headers = {'Content-Type' => 'text/html'}
-    @body = ''
+    @headers = {'Content-Type' => 'application/json'}
   end
 
-  def status value = nil
-    value ? @status = value : @status
+  def status(value = nil)
+    value ? @status = value : @status || 200
```

Comment:
ok so there are some response headers but no response body. well we can dig around in the code for anything that might short-cut to an empty response being sent by accident. are there unit tests to help isolate the issue?